optimo/models.py

```diff
@@ -382,6 +396,18 @@ def validate(self):
                     .format(cls_name, 'ServiceRegionPolygon')
                 )
 
+        if self.cost_per_hour is not None:
+            self.validate_type('cost_per_hour', Number)
+
+        if self.cost_per_hour_for_overtime is not None:
+            self.validate_type('cost_per_hour_for_overtime', Number)
+
+        if self.cost_per_km is not None:
+            self.validate_type('cost_per_km', Number)
```

Comment:
Shouldn't we be something keeping track of the type of units this is expressed in?  I'm guessing different businesses will want to express it in incomparable units.

Reply:
I think this should only be kilometers. The API user can do the conversion.

Reply:
I meant the 'cost' part, not the 'distance' part.

Reply:
Good point, I'll think about that